docs/guide/sending_notifications.md

```diff
@@ -19,7 +19,7 @@ There are four types of notifications: `info`, `success`, `warning` and `error`.
 
 ### Display notifications from PHP
 
-To send a notification from PHP, inject the `TranslatableNotificationHandlerInterface` into your class.
+To send a notification from PHP, inject the [`TranslatableNotificationHandlerInterface`](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/Notification/TranslatableNotificationHandler.php) into your class.
```

Comment:
we do not link to master as it will change overtime, please specify a version of admin-ui bundle

Reply:
For the time being, I would leave the link leading to the master version as I'm going to merge this PR to the master version of our documentation. When it comes to all the remaining versions of the doc , I will align the links to the file as you suggested. :)